lib/rust/ensogl/core/src/animation/loops.rs

```diff
@@ -204,28 +212,52 @@ pub fn on_before_rendering() -> enso_frp::Sampler<TimeInfo> {
 #[derivative(Debug(bound = ""))]
 pub struct LoopRegistry {
```

Comment:
I noticed the docs must be updated, because they say about `An animation loop`, but this is names `registry`.